titration/ui_state/demo_mode/demo_pump.py

```diff
@@ -35,18 +36,25 @@ def handle_key(self, key):
         if self.substate == 1:
             if key == Keypad.KEY_1:
                 self.substate = 3
+
             elif key == Keypad.KEY_2:
                 self._set_next_state(PumpVolume(self.titrator, self), True)
+                self.substate = 3
+
             elif key == Keypad.KEY_3:
-                # self.titrator.pump.pump_volume_in()
-                pass
+                self._set_next_state(VolumeToMove(self.titrator, self), True)
+                self.titrator.pump.pull_volume_in(self.titrator.volume_to_move)
+                self.titrator.volume_to_move = 0
```

Comment:
Does this go to the view volume state after collecting the input?

Reply:
It does now. This issue has been fixed.